src/sentry/static/sentry/app/views/groupDetails/seenBy.jsx

```diff
@@ -52,7 +52,9 @@ const GroupSeenBy = React.createClass({
     return (
       <div className="seen-by">
         <ul>
-          <li><span className="icon-eye" /></li>
+          <li>
+            <span className="icon-eye tip" title="People who've viewed this issue" />
```

Comment:
It would be good to make sure we translate this. We have a function that handles this so you can at the top of the file you can do
`import {t} from '../../locale';`
and then your span will be
```
            <span className="icon-eye tip" title={t("People who've viewed this issue")} />
```
This allows the text to translate to other languages based on browser language settings I think.